### PR TITLE
chore: disable some unused variable warnings

### PR DESCRIPTION
**Pull request by patricebender**
its annoying that those show up in every pull request, so lets ignore those lines

### CODE REVIEW

File: db-service/test/cds-infer/elements.test.js [hunk 1]
```diff
@@ -201,6 +201,7 @@ describe('infer elements', () => {
       let { Books, Receipt } = model.entities
       expect(inferred).to.have.nested.property('sources.Books', Books)
       expect(inferred).to.have.nested.property('sources.Receipt', Receipt)
+      // eslint-disable-next-line no-unused-vars
       const { image, ...BooksElementsWithoutBlob } = Books.elements
```
Comment (David-Kunz) [inline db-service/test/cds-infer/elements.test.js:205]:
```suggestion
      const { image: _image, ...BooksElementsWithoutBlob } = Books.elements
```
That should also work.

Comment (David-Kunz) [reply db-service/test/cds-infer/elements.test.js:205]: Reason: ESLint allows unused variables if they start with an underscore.

Comment (patricebender) [reply db-service/test/cds-infer/elements.test.js:205]: Thanks David!

File: db-service/test/cds-infer/elements.test.js [hunk 2]
```diff
@@ -201,7 +201,7 @@ describe('infer elements', () => {
       let { Books, Receipt } = model.entities
       expect(inferred).to.have.nested.property('sources.Books', Books)
       expect(inferred).to.have.nested.property('sources.Receipt', Receipt)
-      const { image, ...BooksElementsWithoutBlob } = Books.elements
+      const { image: _image, ...BooksElementsWithoutBlob } = Books.elements
```
Comment (patricebender) [inline db-service/test/cds-infer/elements.test.js:204]: @David-Kunz renaming the variable has no effect :(

Comment (patricebender) [inline db-service/test/cds-infer/elements.test.js:204]:
```suggestion
       // eslint-disable-next-line no-unused-vars
      const { image, ...BooksElementsWithoutBlob } = Books.elements
```

**Review by stewsk — APPROVED** [review]

**Pull request merged**